docker-compose.sub.yml

```
@@ -0,0 +1,43 @@
+      # 알림
+      - SLACK_WEBHOOK_URL=${SLACK_WEBHOOK_URL}
+      - FCM_BASE64_KEY=${FCM_BASE64_KEY}
+
+      # AWS S3
+      - AWS_ACCESS_KEY=${AWS_ACCESS_KEY}
+      - AWS_SECRET_KEY=${AWS_SECRET_KEY}
+      - S3_BUCKET=${S3_BUCKET}
+    expose:
+      - "8080"
+    healthcheck:
+      test: [ "CMD-SHELL", "wget --quiet --tries=1 --spider http://localhost:8080/actuator/health || exit 1" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)